src/archive.rs

```diff
@@ -254,6 +271,37 @@ impl OpenArchive {
         }
     }
 
+    pub fn is_locked(&self) -> bool {
+        self.flags.contains(ArchiveFlags::LOCK)
+    }
+
+    pub fn has_encrypted_headers(&self) -> bool {
+        self.flags.contains(ArchiveFlags::ENC_HEADERS)
+    }
+
+    pub fn has_recovery_record(&self) -> bool {
+        self.flags.contains(ArchiveFlags::RECOVERY)
+    }
+
+    pub fn has_comment(&self) -> bool {
+        self.flags.contains(ArchiveFlags::COMMENT)
+    }
+
+    /// Solid archive, all files in a single compressed block.
+    pub fn is_solid(&self) -> bool {
+        self.flags.contains(ArchiveFlags::SOLID)
+    }
+
+    /// Archive is part of a multivolume set, but not the first volume.
+    pub fn is_volume(&self) -> bool {
```

Comment:
This method name might cause some confusion if someone's running it on the first volume.
Yes, you are documenting it, but maybe it would make more sense to add an abstraction over these raw flags. Maybe give this method a semi-obscure or uncofusable name (like `is_non_first_volume` or `is_proceeding_volume` or `is_subsequent_volume` or `is_sequel`) and in addition add a method like `pub fn volume_info(&self) -> VolumeInfo` and define it as `enum VolumeInfo { None, First, Subsequent }`

Reply:
You're right, I'll go with the `VolumeInfo` approach.